src/screens/TermsAndConditions/index.js

```diff
@@ -0,0 +1,163 @@
+          suitability of the information and materials found or offered on this
+          website for any particular purpose. You acknowledge that such
+          information and materials may contain inaccuracies or errors and we
+          expressly exclude liability for any such inaccuracies or errors to the
+          fullest extent permitted by law.
+        </Text>
+        <Text style={styles.tcL}>
+          {'\u2022'} Your use of any information or materials on this website is
+          entirely at your own risk, for which we shall not be liable. It shall
+          be your own responsibility to ensure that any products, services or
+          information available through this website meet your specific
+          requirements.
+        </Text>
+        <Text style={styles.tcL}>
+          {'\u2022'} This website contains material which is owned by or
+          licensed to us. This material includes, but is not limited to, the
+          design, layout, look, appearance and graphics. Reproduction is
+          prohibited other than in accordance with the copyright notice, which
+          forms part of these terms and conditions.
+        </Text>
+        <Text style={styles.tcL}>
+          {'\u2022'} All trademarks reproduced in this website, which are not
+          the property of, or licensed to the operator, are acknowledged on the
+          website. Unauthorised use of this website may give rise to a claim for
+          damages and/or be a criminal offence.
+        </Text>
+        <Text style={styles.tcL}>
+          {'\u2022'} From time to time, this website may also include links to
+          other websites. These links are provided for your convenience to
+          provide further information. They do not signify that we endorse the
+          website(s). We have no responsibility for the content of the linked
+          website(s).
+        </Text>
+        <Text style={styles.tcL}>
+          {'\u2022'} Your use of this website and any dispute arising out of
+          such use of the website is subject to the laws of England, Northern
+          Ireland, Scotland and Wales.
+        </Text>
+        <Text style={styles.tcP}>
+          The use of this website is subject to the following terms of use
+        </Text>
+      </ScrollView>
+      <TouchableOpacity
+        disabled={!isRead}
+        onPress={() => navigation.navigate('Onboarding')}
+        style={isRead ? styles.button : styles.buttonDisabled}>
+        <Text style={styles.buttonLabel}>Accept terms and conditions</Text>
+      </TouchableOpacity>
+    </View>
+  );
+}
+
+const {width, height} = Dimensions.get('window');
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 20,
+    marginLeft: 10,
+    marginRight: 10,
+  },
+  title: {
+    fontSize: 22,
+    alignSelf: 'center',
+  },
+  tcP: {
+    marginTop: 10,
+    marginBottom: 10,
+    fontSize: 12,
+  },
+  tcP: {
+    marginTop: 10,
+    fontSize: 12,
+  },
```

Comment:
why do u have two the same items with different styles? what is tcP? i guess it has to be some one convention to call items to style them?

Reply:
will fix the duplicates. tcP is terms and conditions paragraph and tcL is terms and conditions list.